src/tests/test_console.py

```diff
@@ -429,7 +429,7 @@ def __repr__(self):
 
             >>> Test()
             [[;;;terminal-error]Traceback (most recent call last):
-              File \"/lib/python3.9/site-packages/_pyodide/console.py\", line 478, in repr_shorten
+              File \"/lib/python3.9/site-packages/_pyodide/console.py\", line 475, in repr_shorten
                 text = repr(value)
```

Comment:
We really ought to remove this number from the string before making the comparison...

Reply:
Yeah...